config/filesystem.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"github.com/qri-io/jsonschema"
+)
+
+// Filesystem configures the filesystem for qri content
+type Filesystem struct {
```

Comment:
I've tried to compose this struct from `qfs/muxfs.MuxConfig` as @ramfox suggested, however the import was giving me a weird error :/

```
go: finding module for package github.com/qri-io/qfs/muxfs
go: github.com/qri-io/qri imports
        github.com/qri-io/qri/cmd imports
        github.com/qri-io/qri/config imports
        github.com/qri-io/qfs/muxfs: package provided by github.com/qri-io/qfs at latest version v0.1.0 but not at required version v0.1.1-0.20200207165332-a6a7b98f0fa0
```

Reply:
Pretty sure this is because we are currently relying on an older version of qfs.

Maybe we refactor once we can rely on new `qfs`, but I think this is great!